File: CS/LayoutCntMoveInGroup/MyLayoutControl.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.Xpf.LayoutControl;

namespace LayoutCntMoveInGroup
{
    class MyLayoutControl : LayoutControl
    {
        protected override DevExpress.Xpf.Core.PanelControllerBase CreateController()
        {
            return new MyLayoutControlController(this);
        }

    }
}

File: VB/LayoutCntMoveInGroup/MyLayoutControl.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.Xpf.LayoutControl

Namespace LayoutCntMoveInGroup
	Friend Class MyLayoutControl
		Inherits LayoutControl
		Protected Overrides Function CreateController() As DevExpress.Xpf.Core.PanelControllerBase
			Return New MyLayoutControlController(Me)
		End Function

	End Class
End Namespace
